Monika After Story/game/pong.rpy

```diff
@@ -430,28 +579,502 @@ init -1 python in mas_pong:
+                        m 4eka "Drinking a glass of water can also help."
+                        $ player_lets_monika_win_on_purpose = False
+                    
+                    
+        #More than three times
+        else:
+            if player_lets_monika_win_on_purpose:
+                m 5eub "Aren't you getting tired of letting me win, [player]?"#
+            else: 
+                m 1rsc "..."#
+        
+        
+    #Monika wins a game after the player let her win on purpose at least three times
+    elif instant_loss_streak_counter_before >= 3 and player_lets_monika_win_on_purpose:
+        m 3hub "Nice try, [player]!"#
+        m 6eua "As you see I can win all by myself!"
+        m 5hub "Ehehe~"
+        
+    
+    #Monika wins after telling the player she would win the next game
+    elif powerup_value_this_game == DIFFICULTY_POWERUP:
+        m 1hub "Eheheh."
+        
+        $ dt = datetime.date.today()
+        $ dt = datetime.datetime(dt.year, dt.month, dt.day)
+
+        if persistent.pong_difficulty_powerup_date == dt:
+            m 2tsb "Didn't I tell you I would win this time?"#
+        else:
+            m 4tsb "Remember how I told you I would win the next game?"
+    
+    
+    #Monika wins after going easy on the player
+    elif powerup_value_this_game == DIFFICULTY_POWERDOWN:
+        m 2wud "Oh."
+        m 3esb "Try again, [player]!"
+        $ persistent.pong_difficulty_powerup = DIFFICULTY_POWERDOWNBIG;  
+    
+    
+    #Monika wins after going even easier on the player
+    elif powerup_value_this_game == DIFFICULTY_POWERDOWNBIG:
+        m 2lssdrb "Ahahaha."
+        m 2lksdrb "I really hoped you would win this game."
+        m 2hksdrb "Sorry about that, [player]!"
+        
+        
+    #The player has lost 3, 8, 13, ... matches in a row.
+    elif loss_streak_counter >= 3 and loss_streak_counter % 5 == 3:
+        m 2eku "Come on, [player], I know you can beat me!" 
+        m 2duu "Stay determined!"
+        
+        
+    #The player has lost 5, 10, 15, ... matches in a row.
+    elif loss_streak_counter >= 5 and loss_streak_counter % 5 == 0:
+        m 2esd "I hope you are having fun, [player]."
+        m 2rksdlc "I don't want you to feel bad for losing too often."
+        
+        
+    #Monika wins after the player got a 3+ winstreak
+    elif win_streak_counter_before >= 3:
+        m 1hub "Ahahaha!"
+        m 1duu "Sorry, [player]."
+        m 1tub "It looks like your luck has run out."
+        m 4hua "Now it's my turn to shine~"
+            
+        $ pong_last_responce_id = 60
+        
+        
+    #Monika wins a second time after the player got a 3+ winstreak
+    elif pong_last_responce_id == 60:
+        m 6hua "Ehehehe!"
+        m 1eub "Keep up, [player]!"
+        
+        if persistent.gender == "M":
+            m 2tku "You don't want to lose to a girl like that, do you?"
```

Comment:
It might just be me being < a fragile little flower like I can be, but couldn't this be offensive to some people and can this be allowed in?

Might just be me, we'll see.